pm_icecon/util.py

```diff
@@ -40,13 +39,13 @@ def standard_output_filename(
     *,
     hemisphere: Hemisphere,
     date: dt.date,
-    sat: ValidSatellites,
```

Comment:
Loosen our type on `standard_output_filename`. This should remain flexible and the concept of 'valid satellites' needs to change anyway. `pm_icecon` shouldn't need to know about what satellites produced data passed to it. It should be the responsibility of programs calling functions from this code to provide TBs in an expected format.